internal/winapi/zsyscall_windows.go

```diff
@@ -52,11 +53,12 @@ var (
 	procQueryIoRateControlInformationJobObject = modkernel32.NewProc("QueryIoRateControlInformationJobObject")
 	procNtOpenJobObject                        = modntdll.NewProc("NtOpenJobObject")
 	procNtCreateJobObject                      = modntdll.NewProc("NtCreateJobObject")
-	procSearchPathW                            = modkernel32.NewProc("SearchPathW")
 	procLogonUserW                             = modadvapi32.NewProc("LogonUserW")
 	procRtlMoveMemory                          = modkernel32.NewProc("RtlMoveMemory")
 	procLocalAlloc                             = modkernel32.NewProc("LocalAlloc")
 	procLocalFree                              = modkernel32.NewProc("LocalFree")
+	procQueryWorkingSet                        = modpsapi.NewProc("QueryWorkingSet")
+	procGetProcessImageFileNameW               = modkernel32.NewProc("GetProcessImageFileNameW")
```

Comment:
Am I missing something or do we not actually call `QueryWorkingSet` or `GetProcessImageFileName` anywhere?

Reply:
We don't, I defined these for adding the Properties and PropertiesV2 calls to map as closely to what hcs containers return but I decided to leave this for another PR after this.